<launch>




<include file="$(find parameter_tuning)/launch/parameter_tuner.launch">
	<arg name="dynreconfigure_name" value="local_planner_nodelet"/>
	<arg name="image_topic" value="/camera_front/rgb/image_raw"/>
	<arg name="hz" value="10"/>
	<arg name="scale" value="1."/>
	<arg name="config" value="configs/ppliteseg_hannasscapes.yml"/>
	<arg name="labels" value="weights/ppliteseg_hannasscapes/labels.json"/>
	<arg name="device" value="cpu"/>

</include>

</launch>
